packages/xo-web/src/xo-app/backup/file-restore/index.js

```diff
@@ -86,7 +86,11 @@ export default class Restore extends Component {
   }
 
   _refreshBackupList = async (_remotes = this.props.remotes, jobs = this.props.jobs) => {
-    const remotes = keyBy(filter(_remotes, 'enabled'), 'id')
+    const remotes = keyBy(
+      // S3 remote not supported
```

Comment:
I don't like this comment: IMHO the code is clear enough and the server's side condition can change in the future.